docs/tutorials/rerank/rerank_pipeline_with_Bedrock_Rerank_model.md

```diff
@@ -0,0 +1,703 @@
+# Topic
+
+A [reranking pipeline](https://opensearch.org/docs/latest/search-plugins/search-relevance/reranking-search-results/) can rerank search results, providing a relevance score for each document in the search results with respect to the search query. The relevance score is calculated by a cross-encoder model.
+
+This tutorial illustrates using the [Amazon Bedrock Rerank API](https://docs.aws.amazon.com/bedrock/latest/APIReference/API_agent-runtime_Rerank.html) to rerank search results using a model hosted on Amazon Bedrock.
```

Comment:
Can you explain the difference with https://github.com/opensearch-project/ml-commons/blob/main/docs/tutorials/rerank/rerank_pipeline_with_Amazon_Rerank_model_on_Amazon_Bedrock.md ?

I see both can use same model `amazon.rerank-v1:0`. Which tutorial cx should follow ? Any preference ?

Reply:
Previous blueprint requires custom pre and post functions, and calls bedrock invoke API. For invoke API, users need to set model-specific parameters. New blueprint doesn't require custom function code. In addition, new blueprint adopts rerank API. By using Rerank API, users can perform reranking simply by specifying common parameters that are independent of the model. Users can also switch to another model by just changing the model ID.